CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 * Use `zappi/nginx:1.25.3` as the docker base image.
 * Upgrade `headers-more-nginx-module` to `v0.37`.
+* Update latency headers:
+  * Add `X-Proxy-Request-Time`.
+  * Add `X-Proxy-Backend-Response-Time`.
+  * Remove `X-Proxy-Backend-Total-Time`.
+* Update logging fields:
+  * Remove `http_x_proxy_backend_total_time`.
+  * Remove `http_x_proxy_backend_response_time`.
```

Comment:
We don't seem to have removed `http_x_proxy_backend_response_time` from logging fields 🤔 , [here](https://github.com/Intellection/docker-nginx-proxy/pull/19/files#diff-90a4c194ce97fec3a50e6057aac0884d21e83d31dbed855371d66dc0afe83e12R18).

Secondly would we not want to keep the logging and http configuration consistent? 
Meaning adding `http_x_proxy_request_time`.

Reply:
> We don't seem to have removed `http_x_proxy_backend_response_time` from logging fields 🤔 , [here](https://github.com/Intellection/docker-nginx-proxy/pull/19/files#diff-90a4c194ce97fec3a50e6057aac0884d21e83d31dbed855371d66dc0afe83e12R18).

Right, fixed in https://github.com/Intellection/docker-nginx-proxy/pull/19/commits/977c88b0863bb4e5f046f499ea79b0486f337d75.

> Secondly would we not want to keep the logging and http configuration consistent? Meaning adding `http_x_proxy_request_time`.

Turns out we already do ...

https://github.com/Intellection/docker-nginx-proxy/blob/facd8e70bae39b13daf10c2b9d60ab847834f0be/config/log.conf#L26-L28